plugins/linux-v4l2/v4l2-controls.c

```diff
@@ -99,7 +98,12 @@ static int_fast32_t v4l2_update_controls_menu(int_fast32_t dev,
 		}
 	}
 
-	return 0;
+	if (obs_property_list_item_count(prop) == 0) {
```

Comment:
Can we tell this from the `qctrl` properties or is it the case that v4l2_ioctl for all the qctrl's reported is non-zero?

Reply:
Unfortunately, I don’t see any way to determine this other than by iterating through each potential value and checking it with `QUERYMENU`. On the device I tested with, `qctrl->flags` is `0`, and nothing else indicates anything out of the ordinary as far as I can tell. I even peeked a little in the UVC driver itself, and it seems like a number of the values (such as `minimum`, `maximum`, and `step`) are entirely hardcoded, so there’s no help there.

Reply:
Thanks for taking a look.